source/triggers/groups/triggerGroup.js

```diff
@@ -13,41 +13,41 @@
 // limitations under the License.
 
 this.initialize = function() {
-    this.triggers = [];
+    this.triggers$ = [];
     this.canFire$ = [];
-    this.checkFrequency$ = 0.08 + ( Math.random() * 0.02 );
-    // this.future( this.checkFrequency$ ).checkTriggers$();
+    this.checkFrequency$ = 0.04 + ( Math.random() * 0.02 );
+    this.future( this.checkFrequency$ ).checkTriggers$();
```

Comment:
It's really easy to test that it's working (after you remove it) - if triggers work at all, you're good.

We might add a comment on the checkFrequency$ function, saying where it first gets called from, since that's now non-obvious.